docs/excel/configure-your-add-in-to-use-the-browser-runtime.md

```diff
@@ -0,0 +1,114 @@
+---
+ms.date: 02/06/2020
+title: "Configure your Excel add-in to share the browser runtime (preview)"
+ms.prod: excel
+description: Configure your Excel add-in to share the browser runtime and run ribbon, task pane, and custom function code in the same runtime.
+localization_priority: Priority
+---
+
+# Configure your Excel add-in to share the browser runtime (preview)
+
+[!include[Running custom functions in browser runtime note](../includes/excel-shared-runtime-preview-note.md)]
+
+When running Excel on Windows or Mac, your add-in will run code for ribbon buttons, custom functions, and the task pane in separate JavaScript runtime environments. You can configure your Excel add-in to share code in a single browser runtime. This enables better coordination across your add-in for sharing code and data. It also enables you to run code when the document opens, or to run code while the task pane is closed. To configure your add-in to share the browser runtime, follow the instructions in this article.
```

Comment:
The way its written now, the first 2 sentences of this paragraph are almost contradictions. Perhaps you could begin the first sentence with "By default, when ..." and/or begin the second sentence with "However, ..."